src/main/resources/langs/en_US.yml

```diff
@@ -552,8 +552,7 @@ music:
     RepeatMusicQueueCommand:
         error: 'There is nothing to repeat, request music first with `{0}play`'
         success: 'Music queue looping has been turned `:status`.'
-        enabled: 'ON'
```

Comment:
These strings will have to be re-added back at some point, maybe not exactly like they are there, but in a way where a user doesn't get a mix of English and Russian when they enable `Single` for example.

Reply:
So I changed the success info in one of my commits, would that work fine for you?

Reply:
I think it would be better if each state had their own success message, that would be more flexibility when translating the messages, and also doesn't produce a message like:

	Musik spilleren er sat til at gentage `ONE`

Which is a mix of Danish and English, you don't really have to worry about message outputs until everything else is finished anyways, otherwise, you might have to change some of the messages again if you come up with some other implementation.